javascript/chronometer.js

```diff
@@ -1,34 +1,44 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.setIntervalId = setInterval(() => {
+      this.currentTime++
+
+
```

Comment:
Gute Arbeit im ersten Teil, Marten! Hier in der start Funktion musst du später an die callback function denken. Bislang machst du noch nichts mit der callback Funktion.